core/server/api/index.js

```diff
@@ -103,7 +102,8 @@ cacheInvalidationHeader = function cacheInvalidationHeader(req, result) {
             if (hasStatusChanged || wasPublishedUpdated) {
                 return INVALIDATE_ALL;
             } else {
-                return urlService.utils.urlFor({relativeUrl: urlService.utils.urlJoin('/', config.get('routeKeywords').preview, post.uuid, '/')});
+                // routeKeywords.preview: 'p'
```

Comment:
I like this comment/hint for now. We can look up the hardcoded keywords in the code base if we need to 👍

Reply:
Exactly my thought here!